### DISCUSSION TITLE
SO Production Deployment Thought Process

### DISCUSSION THREAD

**Discussion by SecurityStudent1** [Security-Onion-Solutions/securityonion#2026] [Unsupported Versions] [answered]
I'm currently using SO in a lab but I got into a thought experiment about how production deployments would work. Say you wanted to use SO to monitor 2 separate organizations and send all the data back to 1 central management server. Is there a way to differentiate the organization data (like tag traffic) or is this idea not even recommended?

**Comment by dougburks** [comment, marked as answer]
If you deploy a separate forward node for each organization, then each forward node tags the data with its hostname. This allows you to filter events based on what forward node they came from.
